From 382efe4f9720635ed76da0005ceb5465dafdb886 Mon Sep 17 00:00:00 2001
From: Timofei Zhakov <tima@chemodax.net>
Date: Sun, 8 Feb 2026 11:36:45 +0000
Subject: [PATCH] Fix version placeholder in xdigest.pc.in

Ensures the pkgconfig files contains a version. XDIGEST_VERSION is not set
CMakeConfig.txt. Uses XDIG_VERSION.

Closes: #10
Patch by: @andreasstieger


git-svn-id: https://svn.rinrab.com/rinrab/xdigest/trunk@6541 0083e6b4-6090-234a-836e-88fcc467669c
---
 build/xdigest.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/xdigest.pc.in b/build/xdigest.pc.in
index f81dc90..89e3ca1 100644
--- a/build/xdigest.pc.in
+++ b/build/xdigest.pc.in
@@ -4,7 +4,7 @@ includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@/xdigest
 
 Name: libxdigest
 Description: Extremly fast digest algorithms
-Version: @XDIGEST_VERSION@
+Version: @XDIG_VERSION@
 Requires:
 Libs: -L${libdir} -lxdigest
 Cflags: -I${includedir}
